scripts/setup-ubuntu.sh

```diff
@@ -184,6 +185,17 @@ function install_duckdb {
   fi
 }
 
+function install_stemmer {
+  wget_and_untar https://snowballstem.org/dist/libstemmer_c-${STEMMER_VERSION}.tar.gz stemmer
+  (
+    cd stemmer
```

Comment:
This must be `cd ${DEPENDENCY_DIR}/stemmer` since https://github.com/facebookincubator/velox/pull/10920 landed.